server/internal/usecase/interactor/project_test.go

```diff
@@ -167,14 +159,127 @@ func TestProject_Create(t *testing.T) {
 	assert.Nil(t, got)
 
 }
+func TestProject_createProject(t *testing.T) {
```

Comment:
Remove this, not needed.

Reply:
@pyshx 
I remove `TestProject_Create`
`ImportProjectData` and `Create` use this method.